utils/mysql-queries/getRandomKey.js

```diff
@@ -6,10 +8,10 @@ export async function getRandomKey(mysql) {
         FROM ocr_keys
         WHERE
             timeout IS NULL
-            OR timeout < DATE_ADD(NOW(), INTERVAL 55 MINUTE)
+            OR timeout < DATE_ADD(NOW(), INTERVAL ? MICROSECOND)
         ORDER BY
             RAND()
         LIMIT 1
-    `);
+    `, [OCR_SPACE_403_DELAY]);
```

Comment:
Nah, DSL uses it via a question mark a few lines above:
```
INTERVAL ? MICROSECOND
```
Later I would like to move it to the .env file because I want some (this one, at least) variables to be available in grafana (there is a plugin to load .env).

Reply:
You are right, I was too tired when I was reviewing this :slightly_smiling_face: